deployment/network-operator/templates/nodefeaturerules.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.nfd.nodefeaturerules }}
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: nvidia-nics-rules
+spec:
+  rules:
+    - name: "Nvidia NICs PCI"
+      labels:
+        "pci-15b3.present": "true"
+      matchFeatures:
+        - feature: pci.device
+          matchExpressions:
+            vendor: {op: In, value: ["15b3"]}
+            class: {op: In, value: ["0200", "0207"]}
+    - name: "Nvidia NICs SRIOV"
+      labels:
+        "pci-15b3.sriov.capable": "true"
```

Comment:
i dont think we are using it afaik. (also did a quick search in the code)

@e0ne you know ?

Reply:
We don't use it bevause by default NVIDIA nics could have SRIOV disabled in the FW and this selector won't work. but in general It's good to have it to be less vendor-speciffic if possible

Reply:
we still label if its nvidia NIC (15b3) so its still vendor specific.

if we dont need it, lets remove this rule

Reply:
ACK, removing.